### PR TITLE
Replace "Ctrl" by "Cmd" under macOS and style the "kbd" tag in keyboard shortcuts labels

### PR DESCRIPTION
**Pull request by yohanboniface**

### CODE REVIEW

File: umap/static/umap/js/umap.core.js [hunk 1]
```diff
@@ -433,6 +433,64 @@ L.U.Keys = {
...
+    },
+  },
+
+  displayLabel: function (action, withKbdTag=true) {
+    let {shortcut, label} = this.SHORTCUTS[action]
+    if (withKbdTag) {
+      shortcut = shortcut.split('+').map((el) => `<kbd>${el}</kbd>`).join('+')
+    }
+    const modifier = this.isMacOS ? 'Cmd' : 'Ctrl'
+    label += ` ${shortcut.replace('Modifier', modifier)}`
```
Comment (davidbgk) [inline umap/static/umap/js/umap.core.js:490]:
```suggestion
    label += ` (${shortcut.replace('Modifier', modifier)})`
```

I think keeping the keyboard shortcut between parenthesis is more readable.

**Review by davidbgk — COMMENTED** [review]
Is that intentional not to put `kbd` tags in that case?

Capture d’écran, le 2023-12-11 à 15 11 13

When/if applied, the CSS line-height/padding for that tag needs to be reduced.

**Comment by yohanboniface** [comment]
> Is that intentional not to put `kbd` tags in that case?

Not intentional, but this box is derived from the buttons, and use the title element, which does not contain the tag of course. I did not put energy on changing that point, because I thought we'd totally change this box to have all the shortcuts, etc. But let's refactor to use something else than the title, you're right.

File: umap/static/umap/js/umap.core.js (hunk 1 above)
Comment (yohanboniface) [reply umap/static/umap/js/umap.core.js:491]: Once the `kbd` tag is styled, I find the parenthesis noisy:

image


vs 

image

What do you think ?

**Review by davidbgk — APPROVED** [review]

**Pull request merged**